examples/tutorials/api/priors.py

```diff
@@ -71,6 +71,9 @@
     UniformPrior,
 )
 from gammapy.utils.check import check_tutorials_setup
+from gammapy.modeling import PriorParameter
+from gammapy.modeling.models import Model, Prior
+from gammapy.modeling.models import PRIOR_REGISTRY
```

Comment:
In this specific example I would not change the imports and leave them at their initial place in the the tutorial